bundles/org.eclipse.ui.ide/src/org/eclipse/ui/views/markers/internal/ContentGeneratorDescriptor.java

```diff
@@ -20,7 +20,9 @@
 import java.util.HashMap;
```

Comment:
I'm guessing `eclipse.platform.ui/bundles/org.eclipse.ui.ide/META-INF/MANIFEST.MF` needs a version bump too?

Reply:
No, it was already bumped for 4.34